### DISCUSSION TITLE
End outpost intersecting Beacon station

### DISCUSSION THREAD

**Discussion by ABrokenPearl** [FakeFishGames/Barotrauma#14831] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I know this issue has been reported before, though I'm willing to provide more info on the save.

I've been playing vanilla singleplayer for a while and on the third trek from inside the Europan Ridge, the ending outpost is spawned inside of a Beacon Station. 
![20241014231733_1](https://github.com/user-attachments/assets/507ee8a5-f340-4a99-b6e8-65df818320af)
While the beacon was possible to complete (with some cutting), 
![20241014225802_1](https://github.com/user-attachments/assets/75750c9e-4185-4b40-a18b-c748bc562d98)
I could not dock to the station and complete the mission, forcing me to go back to the start.
![20241014231721_1](https://github.com/user-attachments/assets/b21f6370-a32a-42f7-a1e2-9029ccf3d745)
![20241014232019_1](https://github.com/user-attachments/assets/479fdd19-cde7-4ecb-ae78-10b9140f8297)

### Reproduction steps

While I'm unsure how the random values in the game are dictated, I used the seed, "p2tJqEsqUWU~" -and no, I'm not joking.
I would attach my savefile but it seems Github doesnt support '.save' files via attachment.

In my playthrough I got to the Europan Ridge in the Dugong, and got to the point my cursor is (shown in image). There, I found the glitched station.
![20241015001528_1](https://github.com/user-attachments/assets/6d413f37-ffed-42f0-8325-25c3cc907f3e)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Did not cause a crash or error.
```

**Comment by Regalis11** [comment]
> I would attach my savefile but it seems Github doesnt support '.save' files via attachment.

It should be possible to attach the save file if you zip it first. Sending us the save file would make this issue a lot easier for us to diagnose and fix, although I think we can probably find the problem level using the map seed too. Thanks and sorry for the inconvenience!

**Reply by ABrokenPearl** [reply]
[Solo Survivor.zip](https://github.com/user-attachments/files/17383199/Solo.Survivor.zip)

**Reply by Regalis11** [reply]
Thank you! I unfortunately still could not reproduce the issue, the beacon station seems to be positioned correctly in the level between "Soggoth Crevasse Mining Facility" and "Arnthor Habitation Outpost". So there must be some other factors that cause this to trigger, although I'm not sure what those could be. But we'll keep the report open in case we'll find some more clues about this or manage to reproduce the issue ourselves! Sorry for the inconvenience.

**Comment by SomeRandomNoobKekeke** [comment]
On new map with "p2tJqEsqUWU~" that outpost isn't settled yet, [save file](https://github.com/user-attachments/files/17377632/End.outpost.intersecting.Beacon.station.zip)
![20241015145320_1](https://github.com/user-attachments/assets/00551632-182c-48c7-96bf-1ecdbb097507)
My guess is that when you traveled through that passage first time game placed a beacon assuming that there's no outpost
And then new outpost was settled and intersected the beacon

**Reply by ABrokenPearl** [reply]
I understand the confusion due to where my ship is located, but the glitched outpost is the one on the bottom, where my mouse cursor hovers over.

**Reply by SomeRandomNoobKekeke** [reply]
Checked that "glitched outpost", there's nothing intersecting on my end, send save file

**Reply by ABrokenPearl** [reply]
[Solo Survivor.zip](https://github.com/user-attachments/files/17383194/Solo.Survivor.zip)

**Reply by SomeRandomNoobKekeke** [reply]
Hmm... still nothing
On this route right?
![Screenshot_2](https://github.com/user-attachments/assets/d765c3fe-1c13-4d6c-8ef9-085a970f34fa)
![20241015215421_1](https://github.com/user-attachments/assets/831a0300-cdc7-43c7-a342-a806786667a9)

**Reply by ABrokenPearl** [reply]
Yeah. It's not even in the savefile?

**Reply by ABrokenPearl** [reply]
When I'm free I'm going to return to the area and check again if it's there

**Reply by SomeRandomNoobKekeke** [reply]
Yes in your savefile
and there's no mods in that save file right?

**Reply by SomeRandomNoobKekeke** [reply]
> When I'm free I'm going to return to the area and check again if it's there

next time type in console `enablecheats` `debugdraw` `freecamera`
It'll be easier to see

**Reply by ABrokenPearl** [reply]
None, it's first playthrough.
